datm/datm_datamode_cfsr_mod.F90

```diff
@@ -0,0 +1,258 @@
+module datm_datamode_cfsr_mod
+
+  use ESMF
```

Comment:
Please add an only clause here.  CDEPS uses only clause for all module use statements.